app/controllers/concerns/twilio_config.rb

```diff
@@ -46,10 +46,33 @@ def twilio_client
   end
 
   def twilio_outgoing_call_capability
-    @capability ||= Twilio::Util::Capability.new(twilio_creds["account_sid"],
-      twilio_creds["auth_token"])
-    @capability.allow_client_outgoing(twilio_creds["call_app_sid"])
-    @capability
+    account_sid = twilio_creds["account_sid"]
+    api_key = twilio_creds["api_key"]
+    api_secret = twilio_creds["twilio_secret"]
+
+    # Required for Voice
+    outgoing_application_sid = twilio_creds["call_app_sid"]
+    identity = 'user'
+
+    # Create Voice grant for our token
+    grant = Twilio::JWT::AccessToken::VoiceGrant.new
+    grant.outgoing_application_sid = outgoing_application_sid
```

Comment:
can't we directly use `twilio_creds["call_app_sid"]` here?

Reply:
Yes.. we can. Just for readability purpose I added it in separate variable similar to where it is been assigned.